docs/applications/create-commit-statuses-from-application-deployment-statuses.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Create Commit Statuses from Application Deployment Statuses
+intro: Outgoing Webhooks allow users to integrate Devopness to their current CI/CD Workflows. Learn how to create outgoing webhooks to update a commit status check to match your Devopness application deploy pipeline status.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+NOTE: Webhooks, for now, are an API only feature; so this post will guide you through the usage of our API to help you create an outgoing webhook.
+
+NOTE: cURL requests can be imported to API clients, by following the API client's instructions on the links below:
```

Comment:
I believe we should set the context and scenario before tell readers to use an API client:

```suggestion
NOTE: The examples below use `cURL`, which is a command line utility to be used from a terminal window. If you're not familiar with running commands on a terminal or you simply don't want to use a terminal you could, alternatively, copy the examples and import the `curl` command line into your favorite API client platform. Here are links with instructions on how to achieve that on some of the most popular API client platforms:
```

Reply:
Changes applied